transformer_lens/loading_from_pretrained.py

```diff
@@ -721,6 +727,27 @@ def convert_hf_model_config(model_name: str, **kwargs):
             "act_fn": "gelu",
             "attention_dir": "bidirectional",
         }
+    elif architecture == "MistralForCausalLM":
+        cfg_dict = {
+            "d_model": 4096,
+            "d_head": 4096 // 32,
+            "n_heads": 32,
+            "d_mlp": 14336,
+            "n_layers": 32,
+            "n_ctx": 32768,
+            "d_vocab": 32000,
+            "act_fn": "silu",
+            "normalization_type": "RMS",
+            "positional_embedding_type": "rotary",
+            "window_size": 4096,
+            "attn_types": ["local"] * 32,
+            "eps": 1e-05,
+            "n_key_value_heads": 8,
+            "final_rms": True,
```

Comment:
IMO this should be false, final_rms is an override to set _just_ the final normalization to be RMS while the rest is LayerNorm, and isn't needed when it's all RMS